ocular/Cargo.toml

```diff
@@ -14,7 +14,7 @@ readme = "README.md"
 assay = "0.1"
 bech32 = "0.6"
 bip32 = "0.3"
-cosmos-sdk-proto = { version = "0.10", features = ["grpc"] }
+cosmos-sdk-proto = { git = "https://github.com/cosmos/cosmos-rust", branch = "main", features = ["grpc"]}
```

Comment:
We have to read from the latest version of main here as latest release does not have authz or feegrant modules exposed.